smarty/templates/directentry.tpl

```diff
@@ -25,7 +26,51 @@
             {$study_title}
         </th>
     </tr>
-
+        <!-- script src="{$baseurl}/js/loris.js" type="text/javascript"></script -->
+        <script src="{$baseurl}/js/modernizr/modernizr.js" type="text/javascript"></script>
+        <script type="text/javascript">
+          $(document).ready(function() {
+            {if $crumbs != "" && empty($error_message)}
+              var crumbs = {$crumbs|@json_encode},
+                      baseurl = "{$baseurl}",
+                      breadcrumbs = RBreadcrumbs({
+                        breadcrumbs: crumbs,
+                        baseURL: baseurl
+                      });
+              React.render(breadcrumbs, document.getElementById("breadcrumbs"));
+            {/if}
```

Comment:
I don't see the connection between this and a dates polyfill. What's this doing here?  (And in fact, why are breadcrumbs being added to surveys at all?)